pkg/storage/storage.go

```diff
@@ -133,24 +136,27 @@ type ReadUsersetTuplesFilter struct {
 // AuthorizationModelReadBackend Provides a Read interface for managing type definitions.
 type AuthorizationModelReadBackend interface {
 	// ReadAuthorizationModel Read the store type definition corresponding to `id`.
```

Comment:
I know you didn't write or change this, but this is misleading.

Should read something like:
> ... the store model corresponding to `id`

ReadAuthorizationModel doesn't read a typedef, it reads a model.